lib/matplotlib/axes/_axes.py

```diff
@@ -4522,13 +4522,35 @@ def arrow(self, x, y, dx, dy, **kwargs):
         """
         Add an arrow to the axes.
 
-        Call signature::
+        Draws arrow on specified axis from (`x`, `y`) to (`x` + `dx`,
+        `y` + `dy`). Uses FancyArrow patch to construct the arrow.
+
+        Parameters
+        ----------
+        x : float
+            X-coordinate of the arrow base
+        y : float
+            Y-coordinate of the arrow base
+        dx : float
+            Length of arrow in x-coordinate
+        dy : float
+            Length of arrow in y-coordinate
+
+        Return
+        ------
+        a : FancyArrow
+            patches.FancyArrow object
 
-           arrow(x, y, dx, dy, **kwargs)
+        Other Parameters
+        -----------------
+        kwargs : `~matplotlib.patches.FancyArrow` properties.
 
-        Draws arrow on specified axis from (*x*, *y*) to (*x* + *dx*,
-        *y* + *dy*). Uses FancyArrow patch to construct the arrow.
+        See also
+        --------
+        %(FancyArrow)s : Parent object type.
```

Comment:
It turns out this line messes the sphinx rendering. Do you replacing this line as:

         More valid arguments include:
 
         %(FancyArrow)s